libc-test/build.rs

```diff
@@ -221,6 +229,9 @@ fn main() {
             // sighandler_t is crazy across platforms
             "sighandler_t" => true,
 
+            // does not exists on DragonFly
+            "fflags_t" if dragonfly => true,
```

Comment:
Perhaps this could just be removed from the dragonfly definitions? It should be fine to push the typedef down lower into each respective module.